### PR TITLE
PAYEE-197 - add new GDS classes and styles to CY page

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/IncomeTaxSummaryView.scala.html [hunk 1]
```diff
@@ -155,26 +169,26 @@ <h3>@Messages("tai.incomeTaxSummary.annualAmounts.section.totalIncomeHeading")</
...
-                <p class="flush">
-                    <span class="visually-hidden">@Messages("tai.incomeTaxSummary.generalAmount.prefix") </span>
+                <h3 class="govuk-heading-s" >@Messages("tai.incomeTaxSummary.annualAmounts.section.taxFreeHeading")</h3>
+                <p class=" govuk-body flush--bottom">@Messages("tai.incomeTaxSummary.annualAmounts.section.taxFreePara")</p>
+                <p class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold faded-text-dark">
+                    <span class="govuk-visually-hidden">@Messages("tai.incomeTaxSummary.generalAmount.prefix") </span>
                     <span class="display-block heading-36 faded-text-dark section--narrow flush">@viewModel.taxFreeAmount</span>
                 </p>
-                <p class="flush--top">
+                <p class=" govuk-body flush--top">
```
Comment (sergi-hmrc) [inline app/views/IncomeTaxSummaryView.scala.html:178]: nitpick: 👾

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala [hunk 2]
```diff
@@ -60,6 +60,24 @@ trait TaiViewSpec extends BaseSpec with JsoupMatchers {
     }
   }
 
+  def pageWithCombinedHeaderNewFormat(
+    preHeaderText: String,
+    mainHeaderText: String,
+    preHeaderAnnouncementText: Option[String] = None): Unit = {
+    "have an accessible pre heading" in {
+      if (preHeaderAnnouncementText.isDefined) {
```
Comment (sergi-hmrc) [inline test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala:68]: prefer Option methods over if/else:
`preHeaderAnnouncementText.map { txt =>
 doc must havePreHeadingWithTextGds(
          preHeaderText,
          expectedPreHeadingAnnouncement = txt)
}.getOrElse(doc must havePreHeadingWithTextGds(preHeaderText))`

**Review by sergi-hmrc — APPROVED** [review]

File: app/views/IncomeTaxSummaryView.scala.html [hunk 3]
```diff
@@ -45,26 +45,39 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
```
Comment (brendanstraughan) [inline app/views/IncomeTaxSummaryView.scala.html:48]: should this be false, do we not need a backlink?

Comment (masahmad) [reply app/views/IncomeTaxSummaryView.scala.html:48]: no backlink is now handled by the new main template however we still need in the above header for older existing pages

**Review by brendanstraughan — APPROVED** [review]

**Pull request merged**